howto/change-python.md

```diff
@@ -0,0 +1,32 @@
+# Bump Python Version for DjangoGirls.org Website
+
+In order to bump up the Python version for the DjangoGirls.org website, you 
+need to have access to our PythonAnywhere account, and follow these instructions:
+
+1. [Login to PythonAnywhere](https://www.pythonanywhere.com/user/djangogirls2/consoles/). Credentials can be found in 1Password.
+2. Check if the Python version you want to upgrade to is supported by the current
+system image used on PythonAnywhere by clicking on the pen icon on the Web Tab or
+by [reading this](https://help.pythonanywhere.com/pages/ChangingSystemImage#:~:text=Once%20you%20have%20changed%20the,again%20to%20pick%20it%20up.).
```

Comment:
[Deep links to arbitrary content snippets with `#:~:text=` are a Google Chrome specific feature](https://stackoverflow.com/q/62161819/674064) and **don't work in most browsers**. Other browsers only support linking to anchors or to elements with an `id` attribute or similar.

But shouldn't we link to the section listing available Python versions anyway? That's be https://help.pythonanywhere.com/pages/ChangingSystemImage#available-python-versions-for-system-images